content/ru/_index.md

```diff
@@ -0,0 +1,52 @@
+Именно поэтому мы делаем упор на простоту (доступный язык, свободный от заковыристых терминов; примеры, которые понятны любому человеку с базовым пониманием технологий; отказ от лишних подробностей). 
+Работа над глоссарием ведется под руководством подкомитета CNCF по деловой ценности (Business Value Subcommittee, BVS).
+
+<p><img class="mt-3" src="/images/homepage/kubecon.jpg" alt="Зрители смотрят презентацию на KubeCon"></p>
+
+## Внести свой вклад
+
+Приглашаем всех желающих поучаствовать в развитии «Глоссария Cloud Native», его дополнении и улучшении. 
+Процесс разработки единой терминологии и ее совершенствования управляется сообществом и регулируется CNCF. 
+Глоссарий придерживается принципа независимости от конкретных вендоров и ставит своей целью создание словаря-справочника по нативным облачным технологиям.  
+Приветствуется любое участие при условии, что оно соответствует целям и уставу проекта.
+
+Все желающие могут открыть Issue на GitHub или создать Pull Request. 
+Для этого необходимо предварительно ознакомиться с документом [Как внести свой вклад](/ru/contribute/) и присоединиться к каналу [#glossary](https://cloud-native.slack.com/archives/C02TX20MQBB) в рабочем пространстве CNCF в Slack.
+Кроме того, важно убедиться, что предлагаемые правки соответствуют [Руководству по стилю](/ru/style-guide/). 
+Для тех, кто хочет поучаствовать в переводе глоссария на родной язык, существует канал [#glossary-localizations](https://cloud-native.slack.com/archives/C02N2RGFXDF) (для русского языка — [#glossary-localization-russian](https://cloud-native.slack.com/archives/C05G46RMQTX)).
+
+## Благодарности
+
+Глоссарий Cloud Native создан по инициативе маркетингового комитета CNCF (подкомитет по бизнес-ценностям) при участии 
+[Catherine Paganini](https://www.linkedin.com/in/catherinepaganini/en/), 
+[Chris Aniszczyk](https://www.linkedin.com/in/caniszczyk/), 
+[Daniel Jones](https://www.linkedin.com/in/danieljoneseb/?originalSubdomain=uk), 
+[Jason Morgan](https://www.linkedin.com/in/jasonmorgan2/), 
+[Katelin Ramer](https://www.linkedin.com/in/katelinramer/), 
+[Mike Foster](https://www.linkedin.com/in/mfosterche/?originalSubdomain=ca), 
+и многих других авторов. 
+Полный список участников проекта можно найти на [отдельной странице](https://github.com/cncf/glossary/graphs/contributors) на GitHub.
+
+Хранителями глоссария являются:
+- [Noah Ispas](https://www.linkedin.com/in/noah-ispas-0665b42a/); 
+- [Jihoon Seo](https://www.linkedin.com/in/jihoon-seo/);
+- [Nate W.](https://www.linkedin.com/in/nate-double-u/);
+- [Jorge Castro](https://www.linkedin.com/in/jorge-castro2112/).
+
+[Catherine Paganini](https://www.linkedin.com/in/catherinepaganini/en/)
+и [Jason Morgan](https://www.linkedin.com/in/jasonmorgan2/)
+являются Почетными Хранителями, и мы глубоко признательны
+за их неоценимый вклад в развитие проекта на протяжении многих лет.
```

Comment:
```suggestion
за их неоценимый вклад в развитие проекта на протяжении многих лет.

Lastly, the Russian localization of the Cloud Native Glossary was initiated through the contributions of the Russian Localization Team (#glossary-localization-korean), including ..., ..., ...., ..., .., ..., various other contributors.
```

Please add acknowledgement for major localization contributors who participated this first version. :)